### DISCUSSION TITLE
Documentation and error message for character/byte limit conflict

### DISCUSSION THREAD

**Discussion by Bigpet** [community/community#38765] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The create run API documentation here: https://docs.github.com/en/rest/checks/runs#create-a-check-run
mentions for properties like `summary` and `text` in the `output` object: `Maximum length: 65535 characters.`

However when calling this endpoint with octokit with a string that is less than 65535 characters but more than 65535 bytes then the HTTP response contains: 
`{"resource":"CheckRun","code":"custom","field":"summary","message":"summary exceeds a maximum bytesize of 65535"}`

I would like the documentation to be updated or the endpoint to accept the number of characters instead of bytes.

(I also found some places on the web referecing this old post which I can't find anywhere, just redirects me to the community issues homepage: https://github.community/t/undocumented-65535-character-limit-on-requests/117564)

**Comment by ossanna16** [comment]
Hi there @Bigpet and welcome to our community! Thank you for reporting this bug!

**Comment by ncast-dev** [comment]
## SWB-022 — GET /health endpoint livré ✅

**Repo**: switchboard (main)
**Commit**: a2bec85

### Ce qui a été fait
- Endpoint `GET /health` ajouté dans `api/rest.py` (avant le middleware auth)
- `/health` ajouté à la whitelist du middleware (côté auth exclusion)
- 4 tests dans `tests/test_SWB-022.py` : 200 sans token, status ok, version présente, accessible même avec token invalide
- 274 tests existants : tous passent (0 régression)

### Réponse
```json
{"status": "ok", "version": "0.1.0"}
```
Aucune auth requise — accessible pour monitoring externe.

**Reply by ghost** [reply]
tests/test_SWB-022.py

**Reply by Bigpet** [reply]
I don't understand how this is related?

**Comment by ghost** [comment]
https://1.1.1.1/dns-query

**Comment by ghost** [comment]
{"resource":"CheckRun","code":"custom","field":"summary","message":"summary exceeds a maximum bytesize of 65535"}

**Comment by Bigpet** [comment]
This seems to have been fixed in the docs:
```
A short description of the feedback for these lines of code. The maximum size is 64 KB.
```
